src/test/expand.test.ts

```diff
@@ -139,6 +139,10 @@ describe('Expand Abbreviations', () => {
 
   	// https://github.com/microsoft/vscode/issues/114923
 	testExpandWithCompletion('html', 'figcaption', '<figcaption>${0}</figcaption>');
+
+	// https://github.com/microsoft/vscode/issues/72594
```

Comment:
The tests are _mostly_ sorted by issue number, so I would prefer this be placed a bit higher. (Just noticed the figcaption test above has the same issue -- you can move it up too if you want).

Reply:
I've moved both tests to the right place.